newIDE/app/src/MainFrame/index.js

```diff
@@ -1351,6 +1357,11 @@ class MainFrame extends React.Component<Props, State> {
     }
   };
 
+  _onModifiedObject = () => {
```

Comment:
`_onProjectItemModified`?

"Project item" is used in Project Manager to designate something related to the project. "Object" has the meaning in GDevelop of an object in a scene.